commands/cli.go

```diff
@@ -23,24 +23,30 @@ func CreateCli(version string) *cli.App {
 	app.HelpName = app.Name
 	app.Author = "Gruntwork <www.gruntwork.io>"
 	app.Version = version
-	app.Usage = "A CLI tool to cleanup cloud resources (ASG, ELB, ELBv2, EBS, EC2, AMI, Snapshots). THIS TOOL WILL COMPLETELY REMOVE ALL RESOURCES AND ITS EFFECTS ARE IRREVERSIBLE!!!"
-	app.Flags = []cli.Flag{
-		cli.StringSliceFlag{
-			Name:  "exclude-region",
-			Usage: "regions to exclude",
-		},
-		cli.StringFlag{
-			Name:  "older-than",
-			Usage: "Only delete resources older than this specified value. Can be any valid Go duration, such as 10m or 8h.",
-			Value: "0s",
-		},
-		cli.BoolFlag{
-			Name:  "force",
-			Usage: "Skip nuke confirmation prompt. WARNING: this will automatically delete all resources without any confirmation",
+	app.Usage = "A CLI tool to cleanup cloud resources (AWS, Azure, GCP). THIS TOOL WILL COMPLETELY REMOVE ALL RESOURCES AND ITS EFFECTS ARE IRREVERSIBLE!!!"
+	app.Commands = []cli.Command{
```

Comment:
Does `cloud-nuke` (without any args) still show the help text?

Reply:
Yeah it does

Reply:
OK, great